src/commands/command_factory.py

```diff
@@ -0,0 +1,27 @@
+from create_command import *
+
+# from set_command import *
+# from get_command import *
+# from delete_command import *
+
+
+class CommandFactory:
+    def __init__(self, input_adaptor):
+        self.input_adaptor = input_adaptor
+        self.available_commands = {"create", "set", "get", "delete"}
+        self.validate()
+        
+    def create(self):
+        cmd = str(self.input_adaptor.command).lower()
```

Comment:
I have a remark here. I think you don't need to make it ```.lower() ``` again because it has been already turned to lower case in the ```parser_adaptor.py```

Reply:
we solved this and we will push it now